lib/presentation/get_it.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:array_names/data/repositories/repository.dart';
+import 'package:array_names/data/repositories/repository_impl.dart';
+import 'package:array_names/data/services/network_service.dart';
+import 'package:get_it/get_it.dart';
+
+final getIt = GetIt.instance;
```

Comment:
get it is not presentation layer, move to diff folder

Reply:
+